### ISSUE TITLE
First setup script fails after changing the time zone

### ISSUE THREAD

**Issue by Jutsuma** [nextcloud/vm#2406]
### Steps To Reproduce

1.Import VM to Hyper-V manager or setup new VM and import the disks (Gen2, safe boot off)
//I tried both, same result
2.Connect to VM
3.Start the VM
4.Login via default user and password
5.Enter default sudo password
//First setup script starts automatically
6.Try to change keyboard layout and time zone (Dell USB Multimedia, German, no compose key | Time zone: Europe, Berlin)
(7.Try suggested solution)

### Expected Result

Time zone being changed and next step being displayed.

### Actual Result

VM fails to download 'temporary-fix-begining' (screenshot 1).
An error is displayed (screenshot 2).
Screenshot 3 shows me following the suggested solution.

The suggested solution does not work due to a deprication warning (see screenshot 3).

### Screenshots, Videos, or Pastebins

![Screenshot 1](https://user-images.githubusercontent.com/28494366/197415432-4bc80802-40b0-4e73-8dd4-5e240375e328.png)
![Screenshot 2](https://user-images.githubusercontent.com/28494366/197415436-efaeec96-de72-46c3-902e-943de19229f7.png)
![Screenshot 3](https://user-images.githubusercontent.com/28494366/197415439-085a1940-80ee-4658-9fa1-05be61c09cee.png)

### Additional Context

It worked the last times I used it, but that's been a month or two ago, where I used the same settings.
Nothing seems to be wrong with my internet connection.
I am a beginner when it comes to Linux and the script stated, that the first time setup is supposed to be run once and not aborted.
I can use 'sudo bash /var/scripts/menu.sh' to "continue" the setup, but I'm not sure it won't break things later on.

I guess, that I am not seeing an obvious solution a more experienced admin would.

### Build Version

Downloaded virtual disks, last changed on 27.02.2022

### Environment

By downloading the VM

### Environment Details

OS: Windows 10 Pro 21H2
Hypervisor: Hyper-V
CPU: Ryzen 5 5600x (4 cores selected for VM)
RAM: 16GB UDIMM ECC @2440MHz (4196MB for VM, dynamic)
VM connection: Default Switch
VM gen: Gen2/UEFI, safe boot off)

**Comment by enoch85** [comment]
Yup, this seems like a bug for sure. Will have a look at this when I have time. Thanks for posting! :+1:

**Comment by enoch85** [comment]
@Jutsuma How old is the VM download? We had issues a while ago with the script downloading, but that was fixed in later versions.

**Comment by enoch85** [comment]
Now when I look closer it's not a bug either, since you should use only *one* ending. So instead of:

`.sh|php|py`
use:
`.sh` *only*

**Comment by Jutsuma** [comment]
@enoch85
About downloading a newer version:
Well it is a paid version and only downloadable once in a short time window, I can't just download a new 40GB Hyper-V version.
Do I need an account to always be able to download an up-to date version on hanssonit.se?
If so, do I need to buy another Hyper-V VM?

About the problem:
I did wonder about the weird syntax, but I didn't know what it meant, so I just typed what was shown.
I have tried it with the correct way now, but I don't get how to "try again" since I can reboot all I want, I always get the same error, because the download fails and stops the script.

I am sorry for the absolute beginner incompetency, but I have yet to find the according documentation for that.
I just see how to run the menu script after the first setup. 
Due to the script and documentation being rather clear about the first setup being important, I still don't know whether I'd break something later on if I use 'sudo bash /var/scripts/menu.sh'.
